modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/opml/OpmlImportTask.kt

```diff
@@ -227,6 +240,27 @@ class OpmlImportTask @AssistedInject constructor(
         processUrls(urls)
         return urls.size
     }
+    private fun replaceInvalidXmlCharacter(inputStream: InputStream): InputStream {
+        val content = inputStream.bufferedReader().use { it.readText() }
+
+        val indentedContent = content.replace("<outline", "\n<outline")
+
+        val modifiedContent = indentedContent
+            .replace("&", "&amp;")
```

Comment:
If a file is correctly formatted this breaks it. Imagine that a file has `&amp;` inside of it. It would be changed to `&amp;amp;` or `&gt;` to `&amp;gt;` and so on.